docs/span-api.asciidoc

```diff
@@ -174,3 +174,30 @@ the current time will be used
 End the span.
 If the span has already ended,
 nothing happens.
+
+[[span-outcome]]
+==== `span.outcome`
+[small]#Added in: v3.12.0#
+
+The Node.js agent automatically sets an `outcome` property on spans.  This property will be one of three values
+
+`success`: Indicates the span's operation was a success.
+
+`failure`: Indicates the span's operation was _not_ a success.
+
+`unknown`: Indicates the agent was unable to determine whether the span's operation was a success or not. An `unknown` outcome removes a transaction from error rate considerations.
```

Comment:
For all occurrences of these, you might group them as bullet points instead, and add a colon after "values" on L182.

Reply:
✅ will do